src/server.ts

```diff
@@ -358,12 +366,26 @@ export class Server {
 
       let plugin: Plugin;
       let constructor: AccessoryPluginConstructor;
+
       try {
         plugin = this.pluginManager.getPluginForAccessory(accessoryIdentifier);
+      } catch (error) {
+        log.error(`No plugin was found for the accessory "${accessoryIdentifier}" in your config.json at position ${index + 1}. Please make sure the plugin is installed correctly.`);
```

Comment:
This will silence the other possible errors why `getPluginForAccessory` could fail.
Specifically:
```ts
throw new Error(`The requested accessory '${accessoryIdentifier}' has been registered multiple times. Please be more specific by writing one of: ${options}`);
```

This happens when there are multiple plugins having the same accessory/platform name. If that's the case the user must prepend the accessory or platform identifier with the plugin name. As this message is suppressed by this catch clause, the user wouldn't be able to identify the issue.